schemas/com.snowplowanalytics.monitoring.batch/load_succeeded/jsonschema/2-0-1

```diff
@@ -0,0 +1,135 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a data load succeeding",
+  "self": {
+    "vendor": "com.snowplowanalytics.monitoring.batch",
+    "name": "load_succeeded",
+    "format": "jsonschema",
+    "version": "2-0-1"
+  },
+
+  "type": "object",
+  "properties": {
+    "shredding": {
+      "description": "Information about the batch taken from transformation step, isomorphic to shredding_complete schema",
+      "type": "object",
+      "properties": {
+        "base": {
+          "description": "Blob storage path to the root of the batch",
+          "type": "string",
+          "format": "uri",
+          "maxLength": 1024
+        },
+        "compression": {
+          "description": "File compression type",
+          "enum": ["GZIP", "NONE"]
+        },
+        "types": {
+          "description": "Set of Iglu URIs and the format they were shredded into",
+          "type": "array",
+          "items": {
+            "properties": {
+              "schemaKey": {
+                "description": "Iglu URI",
+                "type": "string",
+                "maxLength": 256
+              },
+              "format": {
+                "description": "File format in which the entities were shredded to",
+                "enum": ["TSV", "JSON", "WIDEROW", "PARQUET"]
+              },
+              "shredProperty": {
+                "description": "Field types of enriched event that can be shredded (self-describing JSONs)",
+                "enum": ["SELFDESCRIBING_EVENT", "CONTEXTS"]
+              }
+            },
+            "required": ["schemaKey", "format", "shredProperty"]
```

Comment:
I think if you add a property to the `required` list then you're not allowed to make it a `2-0-1` release because it's not backwards compatible.